packages/create-block/lib/index.js

```diff
@@ -31,22 +33,48 @@ program
 		'template type name, allowed values: "es5", "esnext"',
 		'esnext'
 	)
-	.action( async ( slug, { template } ) => {
+	.option(
+		'--namespace <namespace>',
+		'internal namespace for the block name'
+	)
+	.option( '--title <title>', 'display title for the block' )
+	.option( '--description <description>', 'short description for the block' )
+	.option( '--category <category>', 'category name for the block' )
+	.action( async ( slug, commandObject ) => {
+		const { template } = commandObject;
+		const optionsValues = [
+			'namespace',
+			'title',
+			'description',
+			'category',
+		]
+			.filter( ( optionName ) => commandObject[ optionName ] )
+			.reduce( ( accumulator, optionName ) => {
+				accumulator[ optionName ] = commandObject[ optionName ];
+				return accumulator;
+			}, {} );
```

Comment:
Is this basically equivalent to `pick( commandObject, [ 'namespace', 'title', 'description', 'category' ] )`?

If so, and since we already have Lodash, should it just use `pick`? 

Aside: Can we not spread this object directly to `answers` below, i.e.

```js
const answers = { ...defaultValues, slug, title: startCase( slug ), ...commandObject };
```

I guess it might have extra properties we don't want? Would those break something if included?

Reply:
Yes, I think lodash is one of dependencies so it makes a lot of sense to simplify it to pick.

> I guess it might have extra properties we don't want? Would those break something if included?

Oh yes, it has additional properties. I need to check again because I used console.log to debug quickly. There are help and version properties added by commander, the latter would override another value you can set for the plugin and package.json

Reply:
Yes, I think lodash is one of dependencies so it makes a lot of sense to simplify it to pick.

> I guess it might have extra properties we don't want? Would those break something if included?

Oh yes, it has additional properties. I need to check again because I used console.log to debug quickly. There are help and version properties added by commander, the latter would override another value you can set for the plugin and package.json

Reply:
Quickly tests it with `console.log( Object.keys( commandObject ) );`:

```
$ npx wp-create-block --category layout
[
  'commands',
  'options',
  '_execs',
  '_allowUnknownOption',
  '_args',
  '_name',
  '_optionValues',
  '_storeOptionsAsProperties',
  '_passCommandToAction',
  '_actionResults',
  '_helpFlags',
  '_helpDescription',
  '_helpShortFlag',
  '_helpLongFlag',
  'Command',
  'Option',
  'CommanderError',
  '_description',
  '_argsDescription',
  '_version',
  '_versionOptionName',
  '_events',
  '_eventsCount',
  'template',
  'rawArgs',
  'category',
  'args'
]
```

Reply:
I found an alternative approach: 92440d5.